docs/index.html

```diff
@@ -244,13 +244,15 @@ <h1>Latest Tweets</h1>
 			<h1>Join the Community!</h1>
 			<p>Eclipse SmartHome™ is an open-source project that is driven by
 				its community. Involve yourself now by joining discussions,
-				by reporting issues and by contributing code!</p>
+				by reporting (<a href="https://eclipse.org/security">security related</a>) issues and by contributing code!</p>
 			<a href="https://www.eclipse.org/forums/index.php/f/271/" class="btn waves-effect waves-light green darken-1"
 				type="submit" name="action">Discussion Forum</a>
 			<a href="https://bugs.eclipse.org/bugs/buglist.cgi?list_id=12033781&product=SmartHome&query_format=advanced&resolution=---" class="btn waves-effect waves-light green darken-1"
 				type="submit" name="action">Issue Tracker</a>
 			<a href="https://github.com/eclipse/smarthome" class="btn waves-effect waves-light green darken-1"
 				type="submit" name="action">Git Repository</a>
+			<a href="https://eclipse.org/security" class="btn waves-effect waves-light green darken-1"
```

Comment:
I would not add such a huge button for it, having the link in the text above is imho enough,

Reply:
Ok, but then please see my comment above.